src/python/openvino_genai/py_openvino_genai.pyi

```diff
@@ -0,0 +1,1648 @@
+            :type pad_token_id: int
+        
+            :param translate_token_id: Translate token id.
+            :type translate_token_id: int
+        
+            :param transcribe_token_id: Transcribe token id.
+            :type transcribe_token_id: int
+        
+            :param no_timestamps_token_id: No timestamps token id.
+            :type no_timestamps_token_id: int
+        
+            :param is_multilingual:
+            :type is_multilingual: bool
+        
+            :param begin_suppress_tokens: A list containing tokens that will be suppressed at the beginning of the sampling process.
+            :type begin_suppress_tokens: list[int]
+        
+            :param suppress_tokens: A list containing the non-speech tokens that will be suppressed during generation.
+            :type suppress_tokens: list[int]
+        
+            :param language: Language token to use for generation in the form of <|en|>.
+                             You can find all the possible language tokens in the generation_config.json lang_to_id dictionary.
+            :type language: Optional[str]
+        
+            :param lang_to_id: Language token to token_id map. Initialized from the generation_config.json lang_to_id dictionary.
+            :type lang_to_id: Dict[str, int]
+        
+            :param task: Task to use for generation, either “translate” or “transcribe”
+            :type task: int
+        
+            :param return_timestamps: If `true` the pipeline will return timestamps along the text for *segments* of words in the text.
+                               For instance, if you get
+                               WhisperDecodedResultChunk
+                                   start_ts = 0.5
+                                   end_ts = 1.5
+                                   text = " Hi there!"
+                               then it means the model predicts that the segment "Hi there!" was spoken after `0.5` and before `1.5` seconds.
+                               Note that a segment of text refers to a sequence of one or more words, rather than individual words.
+            :type return_timestamps: bool
+        """
+    def get_generation_config(self) -> WhisperGenerationConfig:
+        ...
+    def get_tokenizer(self) -> Tokenizer:
+        ...
+    def set_generation_config(self, config: WhisperGenerationConfig) -> None:
+        ...
+class draft_model:
+    """
+    This class is used to enable Speculative Decoding
+    """
+    def __init__(self, models_path: os.PathLike, device: str = '', **kwargs) -> None:
+        """
+        device on which inference will be performed
+        """
+EARLY: StopCriteria  # value = <StopCriteria.EARLY: 0>
+HEURISTIC: StopCriteria  # value = <StopCriteria.HEURISTIC: 1>
+NEVER: StopCriteria  # value = <StopCriteria.NEVER: 2>
+NORM_SUM: AggregationMode  # value = <AggregationMode.NORM_SUM: 1>
+SUM: AggregationMode  # value = <AggregationMode.SUM: 0>
```

Comment:
not clear why these values are located inside `openvino_genai` module, while for other enums like `GenerationStatus` - not. 

I would avoid populating such values to main `openvino_genai` module.

Reply:
It's because of `export_values()` line:
https://github.com/openvinotoolkit/openvino.genai/blob/7039c3e3433ff61f1d460fcab674c7b5ca5221eb/src/python/py_continuous_batching_pipeline.cpp#L164

It allows to write
`openvino_genai.SUM, openvino_genai.NORM_SUM` and so on. 
If we remove it, we should use syntax:
`openvino_genai.AggregationMode.SUM, openvino_genai.AggregationMode.NORM_SUM` and so on. 

Removed it.